### DISCUSSION TITLE
Install maturin in Pydroid

### DISCUSSION THREAD

**Discussion by martinklein** [PyO3/maturin#2260] [Q&A]
Maturin is required for openai but there is no package that can be used in Pydroid (at least not in Python 3.11).
I cannot compile it since there seems to be no Rust compiler available in Pydroid.
Many thanks for your help.
Martin Klein

**Comment by messense** [comment]
Sorry but I don't have an Android device, your best chance of getting it fixed is probably asking Pydroid developer for support.
